frontend/app/templates/components/transportation/study-area-map/census-tract-popup/data.hbs

```diff
@@ -0,0 +1,5 @@
+<ThrottleProperty @property={{this.geoid}} as |geoid|>
+  <NeedsAsync @needs={{query "census-tract" (hash filter=(hash geoid=geoid))}} as |censusData|>
```

Comment:
cooooool, interested in your thoughts on this approach to using this addon. it is not something we have used before. I think it makes sense for these super ephemeral data fetches (that still go through the store).